roles/argocd/templates/ingress.j2

```diff
@@ -19,4 +19,10 @@ spec:
               service:
                 name: argocd-server
                 port:
-                  name: https
\ No newline at end of file
+                  name: https
+{% if cm_status.stdout == "Active" %}
+  tls:
+  - hosts:
```

Comment:
Please standardize the indentation for list items in yaml.

Reply:
Fixed